sync/metrics.go

```diff
@@ -11,40 +12,152 @@ import (
 var meter = otel.Meter("header/sync")
 
 type metrics struct {
-	totalSynced      atomic.Int64
-	totalSyncedGauge metric.Float64ObservableGauge
+	totalSynced     atomic.Int64
+	totalSyncedInst metric.Int64ObservableGauge
+
+	syncLoopStarted       metric.Int64Counter
+	trustedPeersOutOfSync metric.Int64Counter
+	laggingHeadersStart   metric.Int64Counter
+
+	subjectiveHead atomic.Int64
+	blockTime      metric.Float64Histogram
+
+	headerReceived time.Time
+	prevHeader     time.Time
+
+	subjectiveHeadInst metric.Int64ObservableGauge
```

Comment:
pls group this with subjectiveHead and group headerRec, prevHeader and blockTime together pls